internal/operands/vm-console-proxy/reconcile_test.go

```diff
@@ -54,31 +56,236 @@ var _ = Describe("VM Console Proxy Operand", func() {
 	})
 
 	It("should return name correctly", func() {
-		name := v.Name()
+		name := operand.Name()
 		Expect(name).To(Equal(operandName), "should return correct name")
 	})
 
 	It("should return functions from reconcile correclty", func() {
-		functions, err := v.Reconcile(mockedRequest)
+		functions, err := operand.Reconcile(&request)
 		Expect(err).ToNot(HaveOccurred(), "should not throw err")
 		Expect(len(functions)).To(Equal(7), "should return correct number of reconcile functions")
 	})
+
+	It("should create vm-console-proxy resources", func() {
```

Comment:
Strictly speaking this is not cleaning up existing code but adding new test code. Could you please put it into a separate commit?

Reply:
Ok, then please use this PR to add new unit tests and you can drop the refactoring changes.

Reply:
Currently some new unit tests depend one refactored functions, `isEnabled` and `getVmConsoleProxyNamespace`. Otherwise, unit tests won't fit there.

Reply:
IMO if a PR gets large, multiple commits help to reduce the cognitive load of reviewing the PR significantly. Too many changes in a single commit make PRs very hard to review.